build/templates/_attributes.py.mako

```diff
@@ -122,6 +122,7 @@ class AttributeEnumWithConverter(AttributeEnum):
                 enum value
         '''
         self._underlying_attribute_enum = underlying_attribute_enum
+        self._attribute_id = underlying_attribute_enum._attribute_id
```

Comment:
Can you make this class and `AttributeEnum` class be derived from `Attribute` class, which has the `_attribute_id` attribute?

Reply:
Thanks for the suggestion, I've updated it in my latest commit.